apis/config/v1beta1/defaults_test.go

```diff
@@ -25,6 +27,7 @@ import (
 	componentconfigv1alpha1 "k8s.io/component-base/config/v1alpha1"
 	"k8s.io/utils/ptr"
 
+	"sigs.k8s.io/kueue/apis/config/v1beta1"
```

Comment:
Why this is needed, we are in the package, adding `v1beta1.` throughout the test makes the diff less readable.